source/community/reactnative/src/components/inputs/choice-set/choice-set-input.js

```diff
@@ -43,9 +43,9 @@ export class ChoiceSetInput extends React.Component {
 		this.label = Constants.EmptyString;
 		this.isRequired = this.payload.isRequired || false;
 		this.placeholder = this.payload.placeholder;
-
+		this.pickerRef = React.createRef();
 		this.state = {
-			selectedPickerValue: this.payload.value,
+			selectedPickerValue: (this.payload.choices.find(choice => choice.value === this.payload.value)) && this.payload.value,
```

Comment:
@hariksingh is this condition to check if the value from payload is actually one of the choices? in that case, the 'find' alone will be enough right?